composer.json

```diff
@@ -17,7 +17,7 @@
     "drupal/console": "^1.6",
     "drush/drush": "^9",
     "nikic/php-parser": "~3",
-    "openeuropa/code-review": "^0.2",
+    "openeuropa/code-review": "^0.3",
```

Comment:
Always use the tilde for dependencies that are on version 0, so that it is forwards compatible. This would be better as `~0.3`.